Configuration/StandardSequences/python/Eras.py

```diff
@@ -44,7 +44,7 @@ def __init__(self):
                            'phase2_timing_layer','phase2_hcal',
                            'trackingLowPU', 'trackingPhase1', 'trackingPhase1QuadProp', 'ctpps_2016', 'trackingPhase2PU140',
                            'tracker_apv_vfp30_2016', 'run2_miniAOD_80XLegacy', 'run2_nanoAOD_92X',
-                           'hcalHardcodeConditions', 'hcalSkipPacker']
+                           'hcalHardcodeConditions', 'hcalSkipPacker', 'run2_miniAOD_94XFall17']
```

Comment:
it would be nice to have this edit in the same place as where it shows up in the master branch